index.bs

```diff
@@ -2287,16 +2287,13 @@ W3C Technical Reports</h3>
 
 		<dt>Registries
 		<dd>
-			[=Working Groups=] can also publish registries
-			in order to document collections of values or other data
-			that have no normative implementation requirements.
-			Registries are generally companion to [=Recommendation Track=] documents
-			which contain the related normative requirements,
-			and are typically published in a separate [=registry report=],
-			although they can also be directly embedded in [=Recommendation Track=] documents.
-			The registry track requires [=wide review=] and [=consensus=]
-			on what the registry will contain and how it will be managed.
-			Once set up, changes to registry entries are lightweight
+			[=Working Groups=] can also publish [=registries=]
+			in order to document collections of values or other data.
+			These are typically published in a separate [=registry report=],
+			although they can also be directly embedded in [=Recommendation Track=] documents
+			as a [=registry section=].
+			Establishing a [=registry=] requires [=wide review=] and [=consensus=],
+			but once set up, changes to registry entries are lightweight
```

Comment:
The phrase "Establishing a registry" is doing a lot of work here: it would be helpful to clarify that establishing a Registry means creating a registry definition, and that the registry entries are conceptually distinct.

Reply:
Hmmm, maybe we should use "defining" instead of "establishing" and then link it to the registry definition term?

Reply:
That would be a good improvement.

Reply:
Done in 2dcd7648be90254db1895e0a19a561f5c89bee3d